src/utils/container-runtime/src/handles.rs

```diff
@@ -30,6 +30,10 @@ impl NetworkHandle {
         }
     }
 
+    pub fn name(&self) -> Option<String> {
+        self.network_name.clone()
```

Comment:
Why allocate memory instead of returning `&str`?

Also notice that `network_name` only becomes `None` after the handle is dropped (or `free` is called), so we can use `unwrap` instead of returning an `Option`, as using network name after freeing the handle should not ever happen.